include/zephyr/net/ocpp.h

```diff
@@ -0,0 +1,257 @@
+/*
+ * Copyright (c) 2024 Linumiz
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file ocpp.h
+ *
+ * @defgroup ocpp_api OCPP library
+ * @since 4.3
+ * @version 0.1.0
+ * @ingroup networking
+ * @{
+ * @brief OCPP Charge Point Implementation
+ *
+ * @note The implementation assumes Websocket module is enabled.
+ *
+ * @note By default the implementation uses OCPP version 1.6.
+ */
+
+#ifndef ZEPHYR_INCLUDE_NET_OCPP_H_
+#define ZEPHYR_INCLUDE_NET_OCPP_H_
+
+#include <stdio.h>
+#include <zephyr/kernel.h>
+#include <zephyr/net/net_ip.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** Max length of string literals e.g idtag */
+#define CISTR50	50
+
+/**
+ * @brief OCPP IdTag authorization status in result to ocpp request
+ *        authorization
+ */
+enum ocpp_auth_status {
+	OCPP_AUTH_INVALID,	/*< IdTag not valid */
```

Comment:
```suggestion
	OCPP_AUTH_INVALID,	/**< IdTag not valid */
```

Fix everywhere, this is causing docs to be missing

https://builds.zephyrproject.io/zephyr/pr/68739/docs/doxygen/html/group__ocpp__api.html#ga4b2b8885d144627d43e724a3889ee1cb